src/registrar/assets/sass/_theme/_base.scss

```diff
@@ -129,3 +129,27 @@ abbr[title] {
 .flex-end {
   align-items: flex-end;
 }
+
+// Only apply this custom wrapping to desktop
+@include at-media(desktop) {
+  .usa-tooltip__body {
+    min-width: 350px;
+    max-width: 350px;
+    white-space: normal;
+    text-align: center;
+  }
+}
+
+@media (min-width: 768px) {
```

Comment:
(Q) Why is this not working with @include at-media(mobile)?

Reply:
I think mobile is 480, tablet is 768

Reply:
Of course, duh